draft-ietf-taps-impl.md

```diff
@@ -650,19 +650,115 @@ The reasonable lifetime for cached performance values will vary depending on the
 
 # Specific Transport Protocol Considerations
 
+Each protocol that can run as part of a Transport Services implementation defines both its API mapping as well as implementation details.
+
+API mappings for a protocol apply most to Connections in which the given protocol is the "top" of the Protocol Stack. For example, the mapping of the `Send` function for TCP applies to Connections in which the application directly sends over TCP. If HTTP/2 is used on top of TCP, the HTTP/2 mappings take precendence.
+
+Each protocol has a notion of Connectedness. Possible values for Connectedness are:
+
+- Unconnected. Unconnected protocols do not establish explicit state between endpoints, and do not perform a handshake during Connection establishment.
+- Connected. Connected protocols establish state between endpoints, and perform a handshake during Connection establishment. The handshake may be 0-RTT to send data or resume a session, but bidirectional traffic is required to confirm connectedness.
+- Multiplexing Connected. Multiplexing Connected protocols share properties with Connected protocols, but also explictly support opening multiple application-level flows. This means that they can support cloning new Connection objects without a new explicit handshake.
```

Comment:
Did you want to write "Multiplexed Connected protocols share properties..." ?  Else this seems grammatically wrong to me?  (but maybe it's just me)